A-4E-C/ExternalFM/FM/FuelSystem2.cpp

```diff
@@ -97,11 +104,9 @@ void FuelSystem2::addFuel( double dm, bool wingFirst )
 			externalTanks += m_fuelCapacity[i] > 15.0;
 		}
 
-		double fracDM = dm / (double)externalTanks;
-
-
 		if ( externalTanks > 0 )
 		{
+			double fracDM = dm / (double)externalTanks;
```

Comment:
Why did you move it inside the if?

Reply:
There was a fix here relating to the external fuel tanks I can't remember the exact details though.

Reply:
Even if very unlikely, this was to avoid a divby0 as it was not used outside this if.
Anyway, I messed up with this PR by modifying the branch after, I will resubmit later.